help/en/package/jmri/jmrit/operations/Operations.shtml

```diff
@@ -260,20 +260,29 @@
     
     <a name="Display"></a>
     <h3>Display and Help</h3>
-    
-    To adjust the display font size or appearance of the JMRI windows, select "Preferences.." found under
-    "Edit" in the main window menu bar. A new window will appear with options to make layout connections, change
-    the "Display" appearance and font size, and many other settings including the "Railroad Name". Most
-    JMRI windows have as part of the menu bar the "Help" option. Select "Window Help.." to get information
-    about the window you're using. 
-    
+
+   To adjust the display font size or appearance of the JMRI windows, select "Preferences..." found
+   under "Edit" in the main window menu bar. The
+   <a href="../../../apps/TabbedPreferences.shtml">JMRI: Preferences Pane</a> will appear with
+   options to make <a
+    href="../../../apps/TabbedPreferences.shtml#Connection">Layout Connections</a>, change the
+   window <a href="../../../apps/TabbedPreferences.shtml#Display">Display</a> appearance and font
+   size, and many other settings including the
+   <a href="../../../apps/TabbedPreferences.shtml#RailroadName">Railroad Name</a>. You can also
+   configure JMRI to automatically open any of the operation windows on
+   <a href="../../../apps/TabbedPreferences.shtml#Startup">Start Up</a> by adding a "Perform action".
+   Most JMRI windows have as part of the menu bar the "Help" option. Select "Window Help.." to get
+   information about the window you're using. 
+   
    <a name="MultipleLayouts"></a>
    <h3>Multiple Layout Support</h3>
-   
-   You can create and manage multiple layouts using OperationsPro&trade;. From the main help menu bar under
-   "Edit" select "Preferences" then "Config Profiles".  Create a new configuration profile for each layout
-   that you want to manage.  The database for each layout is unique and not shared with the other layouts.
 
+   You can create and manage multiple layouts using OperationsPro&trade;. From the main help menu
```

Comment:
Thanks for catching that.

 

Dan

 

From: Egbert Broerse [mailto:notifications@github.com] 
Sent: Tuesday, April 26, 2016 2:15 PM
To: JMRI/JMRI <JMRI@noreply.github.com>
Cc: Daniel Boudreau <daboudreau@hotmail.com>; Author <author@noreply.github.com>
Subject: Re: [JMRI/JMRI] Improve help (#1228)

 

In help/en/package/jmri/jmrit/operations/Operations.shtml <https://github.com/JMRI/JMRI/pull/1228#discussion_r61136524> :

>  
> +   You can create and manage multiple layouts using OperationsPro&trade;. From the main help menu

This is the main menu bar (not 'main help menu bar'); note it is in this place only under Windows and Linux, not OS X.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub <https://github.com/JMRI/JMRI/pull/1228/files/ac585d7eac72b5ba77be6b34ce3d4a26303e1e74#r61136524>   <https://github.com/notifications/beacon/ANw2pwaue-nJKWBowsr3XJ6ArOeRNsJrks5p7lYJgaJpZM4IQNJV.gif>